qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -549,6 +581,274 @@ def __deepcopy__(self, _memo: dict = None) -> "IBMBackend":
+        else:
+            session_id = None
+            max_execution_time = None
+            start_session = False
+
+        log_level = getattr(self.options, "log_level", None)  # temporary
+        try:
+            response = self._api_client.program_run(
+                program_id=program_id,
+                backend_name=backend_name,
+                params=inputs,
+                hgp=hgp_name,
+                log_level=log_level,
+                job_tags=job_tags,
+                session_id=session_id,
+                start_session=start_session,
+                max_execution_time=max_execution_time,
+                image=image,
+            )
+        except RequestsApiError as ex:
+            raise IBMBackendApiError("Error submitting job: {}".format(str(ex))) from ex
+        session_id = response.get("session_id")
+        if self._session:
+            self._session._session_id = session_id
+        try:
+            job = RuntimeJob(
+                backend=self,
+                api_client=self._api_client,
+                client_params=self._service._client_params,
+                job_id=response["id"],
+                program_id=program_id,
+                session_id=session_id,
+                service=self.service,
+            )
+            logger.debug("Job %s was successfully submitted.", job.job_id())
+        except TypeError as err:
+            logger.debug("Invalid job data received: %s", response)
+            raise IBMBackendApiProtocolError(
+                "Unexpected return value received from the server "
+                "when submitting job: {}".format(str(err))
+            ) from err
+        Publisher().publish("ibm.job.start", job)
+        return job
+
+    def _get_run_config(self, program_id: str, **kwargs: Any) -> Dict:
+        """Return the consolidated runtime configuration."""
+        # Check if is a QASM3 like program id.
+        if program_id.startswith(QASM3RUNNERPROGRAMID):
+            fields = asdict(QASM3Options()).keys()
+            run_config_dict = QASM3Options().to_transport_dict()
+        else:
+            fields = asdict(QASM2Options()).keys()
+            run_config_dict = QASM2Options().to_transport_dict()
+        backend_options = self._options.__dict__
+        for key, val in kwargs.items():
+            if val is not None:
+                run_config_dict[key] = val
+                if key not in fields and not self.configuration().simulator:
+                    warnings.warn(  # type: ignore[unreachable]
+                        f"{key} is not a recognized runtime option and may be ignored by the backend.",
+                        stacklevel=4,
+                    )
+            elif backend_options.get(key) is not None and key in fields:
+                run_config_dict[key] = backend_options[key]
+        return run_config_dict
+
+    def open_session(self, max_time: Optional[Union[int, str]] = None) -> ProviderSession:
```

Comment:
we have an `open_session` but no `close_session` ?

Reply:
Added `IBM_Backend.close_session` in addition to `IBM_Backend.cancel_session` in https://github.com/Qiskit/qiskit-ibm-provider/pull/762. This needs to be merged before we can use it here.